contracts/tests/test_strapex_contract.cairo

```diff
@@ -0,0 +1,35 @@
+use starknet::{ContractAddress, contract_address_const, get_caller_address};
+
+use snforge_std as snf;
+use snforge_std::{
+    ContractClassTrait, EventSpy, spy_events
+};
+
+use contract_strapex::strapex_factory::{
+    IStrapexFactory, IStrapexFactoryDispatcher, IStrapexFactoryDispatcherTrait, strapex_factory
+};
+
+use contract_strapex::strapex_contract::{
+    IStrapex, IStrapexDispatcher, IStrapexDispatcherTrait, strapex_contract
+};
+
+fn deploy_factory() -> ContractAddress {
+    
+    let factory_contract = snf::declare('strapex_factory');
+    let child_contract = snf::declare('strapex_contract');
+    let child_contract_felt: felt252 = child_contract.class_hash.into();
+    let mut calldata: Array<felt252> = array![
+        contract_address_const::<'OWNER'>().into(),
+        child_contract_felt,
+        contract_address_const::<'DEPOSIT_TOKEN'>().into()
+    ];
+    let contract_address = factory_contract.deploy(@calldata).expect('Deployment failed');
+
+    contract_address
+}
+
+#[test]
```

Comment:
I'm encountering this issue here, @machuwey can you PTAL:
![image](https://github.com/user-attachments/assets/69b89253-154b-456a-9a8d-9a463dac056a)

Reply:
We have to add 
```
[dev-dependencies]
cairo_test = "2.8.4"
```
as a dependency.

Consider also updating to lastest sn_forge version.